src/isPalindrome.js

```diff
@@ -0,0 +1,5 @@
+export default function isPalindrome(input){
+  let firstString = input.toUpperCase().split('').filter((n)=>n.match(/[A-Za-z0-9]/)).join('')
```

Comment:
if you need a line break here you can do one befor eany of the dots (or all of them if you want)

```
input
  .toUpperCase()
  .split('')
  .filter(n => n.match(/[A-Za-z0-9]/))
  .join('')

```
don't have to do that, but this is a long line and it doesn't feel very easy to read.
I'd also take that () away from the `n` and add some spaces around the `=>`